docs/architecture.md

```diff
@@ -1,280 +1,46 @@
-It is used by the [Mapper](#mapping-state-machine) to map a set of deltas from the [Feeder](#feeder) to each block height.
+### Components
 
-[Package documentation](https://pkg.go.dev/github.com/optakt/flow-dps/service/chain)
+* [Loader](https://pkg.go.dev/github.com/optakt/flow-dps/service/loader) -- Loads checkpoints and retrieves their root trie.
+* [Feeder](https://pkg.go.dev/github.com/optakt/flow-dps/service/feeder) -- Loads trie updates in chronological order.
+* [Chain](https://pkg.go.dev/github.com/optakt/flow-dps/service/chain) -- Reads block data at any given height from the protocol state database.
+* [Mapper](https://pkg.go.dev/github.com/optakt/flow-dps/service/mapper) -- Uses the aforementioned components to build its index.
+* [Indexer](https://pkg.go.dev/github.com/optakt/flow-dps/service/index) -- Exposes a Reader and a Writer which give access to the index database.
 
-### Disk Chain
+## Flow DPS Live
 
-The [Disk Chain](https://pkg.go.dev/github.com/optakt/flow-dps/service/chain#Disk) uses the execution node's on-disk key-value store for the Flow protocol state to reconstruct the block sequence.
+The DPS Live binary fetches data from the Flow Network in two distinct ways:
 
-## Feeder
+* It acts as an [unstaked consensus follower](https://github.com/onflow/flow-go/blob/master/follower/consensus_follower.go#L20-L25), which allows it to have access to the protocol state and be notified when a new block is finalized.
+* It downloads block records from a Google Cloud Storage bucket which is continuously updated by the Flow Network.
 
-The Feeder component is responsible for streaming trie updates to the [Mapper](#mapping-state-machine).
-It outputs a state delta for each requested state commitment, so that the [Mapper](#mapping-state-machine) can follow the sequence of changes to the state trie and attribute each change to a block height.
+All of this information is then mapped into the DPS index, which is used by the DPS API.
 
-[Package documentation](https://pkg.go.dev/github.com/optakt/flow-dps/service/feeder)
+[![Flow DPS Live](./svg/flow_dps_live.svg)](./svg/flow_dps_live.svg)
 
-### Disk Feeder
+### Components
 
-The [Disk Feeder](https://pkg.go.dev/github.com/optakt/flow-dps/service/feeder#Disk) reads trie updates directly from an on-disk write-ahead log of the execution node.
+* [GCPStreamer](https://pkg.go.dev/github.com/optakt/flow-dps/service/cloud) -- Downloads block records from a Google Cloud Storage bucket.
+* [Consensus Tracker](https://pkg.go.dev/github.com/optakt/flow-dps/service/tracker#Consensus) -- Reads from the protocol database from an unstaked consensus follower and from the Execution Tracker.
+* [Execution Tracker](https://pkg.go.dev/github.com/optakt/flow-dps/service/tracker#Execution) -- Reads block records from the GCPStreamer.
```

Comment:
Reads block execution records from the GCP streamer and provides access to the state trie updates contained therein.